src/sst/elements/cramSim/tests/testsuite_default_cramSim.py

```diff
@@ -5,14 +5,36 @@
 
 import os
 import shutil
+import tempfile
 
 
 class testcase_cramSim_Component(SSTTestCase):
+    download_file = None
 
     def setUp(self):
         super(type(self), self).setUp()
+        if self.__class__.download_file is None:
+            self.skipTest("File not downloaded!")
```

Comment:
What happens when running the tests concurrently? setUpClass is skipped so these tests will always be skipped?

Reply:
Yes, that's the current behavior.  The fix is more involved than simply calling `setUpClass()` where `initializeClass()` is called...